src/app/components/itinerary/my-itiniraries/my-itiniraries.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'app-my-itiniraries',
+  templateUrl: './my-itiniraries.component.html',
+  styleUrls: ['./my-itiniraries.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7157930)
```suggestion
  styleUrls: ["./my-itiniraries.component.css"]
```